features/cli/oc_expose.feature

```diff
@@ -5,45 +5,27 @@ Feature: oc_expose.feature
   Scenario: Access app througth secure service and regenerate service serving certs if it about to expire
     Given the master version >= "3.3"
```

Comment:
SInce OCP 3.11 goes into ELP phase in June 2022, detail see https://access.redhat.com/support/policy/updates/openshift_noncurrent. we can leave them be. Over the deadline, we can remove them.

Reply:
My suggestion is, for the parts which are not related to the issue being fixed, if such parts are not necessary to change and do not hurt us, we can keep as are.